src/main/java/de/hs_mannheim/informatik/ct/util/CookieManager.java

```diff
@@ -0,0 +1,151 @@
+/*
```

Comment:
Didn't we manage cookies in a much simpler fashion before this? Can you briefly explain, why this is neccessary, pls?